beluga_demo_nav2/launch/beluga_demo_nav2_launch.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2024 Ekumen, Inc.
```

Comment:
@Alondruck is `beluga_demo_nav2_launch.py` a duplicate of `beluga_demo_nav2.launch.py`?

Reply:
Apparently I forgot to delete beluga_demo_nav2.launch.py